atoma-proxy/src/server/handlers/mod.rs

```diff
@@ -82,6 +82,28 @@ pub fn update_state_manager(
     Ok(())
 }
 
+pub fn update_state_manager_fiat(
```

Comment:
This function is never called, I imagine we want to perform this once the AI service request ahas been performed